packages/components/src/sites-table/use-sites-table-sorting.tsx

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line no-restricted-imports
+import type { SiteExcerptData } from 'calypso/data/sites/site-excerpt-types';
+
+interface SitesTableSortOptions {
+	sort?: string;
+	order?: string;
```

Comment:
If we only support two possible orderings, then this is a good way to document and enforce that. It will probably make one of the unit tests (and maybe the `switch` statement?) redundant because Typescript will know those things aren't possible.

```suggestion
	order?: 'asc' | 'desc';
```

Reply:
I adjusted it to define two allowed options. When I try to set order to not supported option, I get a TS warning in IDE. However, it doesn't break compilation, and it doesn't throw any error in the browser console. Do you think it would be wise to leave the test in place in this case?

Reply:
Personally I wouldn't test using a value that TS is already warning about, but I don't think there's anything necessarily wrong with it. _Technically_ it's still possible that an unsupported could slip in at runtime.
But I think the whole point of TS is to prohibit a whole range of possibilities so that you don't have to think about or test for them. For instance we don't write unit tests so that we get well defined behaviour when `sortOrder` is a number or an `instanceof MyClass`. And I think of an invalid string (that TS can check for) as just another such invalid argument.

Reply:
It makes sense, I've removed it then.